src/translations/fr/exportParamsFr.ts

```diff
@@ -0,0 +1,147 @@
+    'iidm.export.xml.extensions.referenceTerminals': 'Terminaux référence de phase (Calcul de répartition)',
+    'iidm.export.xml.extensions.secondaryVoltageControl': 'Réglage secondaire de tension',
+    'iidm.export.xml.extensions.slackTerminal': 'Nœud bilan',
+    'iidm.export.xml.extensions.standbyAutomaton': 'Automate des CSPRs',
+    'iidm.export.xml.extensions.startup': 'Coût de démarrage des groupes',
+    'iidm.export.xml.extensions.substationPosition': 'Coordonnées géographiques des sites',
+    'iidm.export.xml.extensions.threeWindingsTransformerFortescue':
+        'Données pour les calculs dissymétriques des transformateurs à trois enroulements',
+    'iidm.export.xml.extensions.threeWindingsTransformerAsymmetrical':
+        'Données pour les calculs dissymétriques des transformateurs à trois enroulements',
+    'iidm.export.xml.extensions.threeWindingsTransformerPhaseAngleClock':
+        "Angles de phase entre les enroulements (sous forme d'horloge) pour les transformateurs à trois enroulements",
+    'iidm.export.xml.extensions.threeWindingsTransformerToBeEstimated':
+        'Estimation des prises des régleurs et des déphaseurs des transformateurs à trois enroulements',
+    'iidm.export.xml.extensions.twoWindingsTransformerFortescue':
+        'Données pour les calculs dissymétriques des transformateurs à deux enroulements',
+    'iidm.export.xml.extensions.twoWindingsTransformerAsymmetrical':
+        'Données pour les calculs dissymétriques des transformateurs à deux enroulements',
+    'iidm.export.xml.extensions.twoWindingsTransformerPhaseAngleClock':
+        "Angle de phase entre les enroulements (sous forme d'horloge) pour les transformateurs à deux enroulements",
+    'iidm.export.xml.extensions.twoWindingsTransformerToBeEstimated':
+        'Estimation des prises des régleurs et des déphaseurs des transformateurs à deux enroulements',
+    'iidm.export.xml.extensions.voltageLevelShortCircuits': 'Données de court-circuit des postes (Version IIDM 1.0)',
+    'iidm.export.xml.extensions.voltagePerReactivePowerControl':
+        'Lien entre la tension de consigne et la puissance réactive en mode réglage de tension (CSPRs)',
+    'iidm.export.xml.sorted': 'Trier les ouvrages dans le fichier',
+    'iidm.export.xml.sorted.desc': 'Trier les ouvrages dans le fichier',
+    'iidm.export.xml.version': 'Version IIDM',
+    'iidm.export.xml.version.desc': 'Version IIDM utilisée pour générer le fichier',
+    'iidm.export.xml.version.1.0': '1.0',
+    'iidm.export.xml.version.1.1': '1.1',
+    'iidm.export.xml.version.1.2': '1.2',
+    'iidm.export.xml.version.1.3': '1.3',
+    'iidm.export.xml.version.1.4': '1.4',
+    'iidm.export.xml.version.1.5': '1.5',
+    'iidm.export.xml.version.1.6': '1.6',
+    'iidm.export.xml.version.1.7': '1.7',
+    'iidm.export.xml.version.1.8': '1.8',
+    'iidm.export.xml.version.1.9': '1.9',
+    'iidm.export.xml.version.1.10': '1.10',
```

Comment:
The english version is only 132 lines long. Many are missing like `iidm.export.cgmes.export-power-flows-for-switches.desc`.

Is it intentional ?

Reply:
i ve checked, there is nothing missing. The french version is only longer because translations take 2 lines sometimes due to EsLint Prettier

Reply:
Indeed sorry. I got confused.